src/styles/style.css

```
@@ -0,0 +1,347 @@
+body * {
+  font-family: "Nunito", sans-serif;
+}
+
+body {
+  min-height: 100vh;
+}
+
+header, body, main, #stars-container {
+  background-color: var(--bs-light) !important;
+}
+
+h1 {
+  margin: 0 !important;
+}
+
+.navbar-toggler,
+.navbar-toggler:focus {
+  border: none !important;
+  box-shadow: none !important;
+}
+
+#stars-container {
+  padding: 0 20px;
+  height: 50px;
+  background-color: var(--bs-light);
+  display: flex;
+  justify-content: flex-end;
+  overflow: hidden;
+  gap: 3px;
+}
+
+.star {
+  width: 35px;
+  height: 45px;
+  font-size: 30px;
+}
+
+.emoji {
+  font-size: 100px;
+}
+
+.nav-item {
+  cursor: pointer;
+  font-size: medium;
+  border-radius: 30px;
+}
+
+.nav-item:hover {
+  background-color: var(--bs-gray-400);
+  transition: background 0.8s;
+}
+
+.current-nav-item {
+  border-bottom: 2px solid var(--bs-gray);
+  background-color: var(--bs-primary);
+  color: var(--bs-white);
+}
+
+.current-nav-item:hover {
+  background-color: var(--bs-info);
+}
+
+.nav-item.home::before {
+  content: "🏠 ";
+}
+
+#nav-stats::before {
+  content: "📊 ";
+}
+
+.offcanvas {
+  max-width: 400px !important;
+}
+
+.col {
+  perspective: 1000px;
+}
+
+.card {
+  min-width: 150px;
+  max-width: 280px;
+  height: 220px !important;
+  position: relative;
+  transition: 0.7s;
+  transform-style: preserve-3d;
+}
+
+.card:hover {
+  box-shadow: rgba(17, 17, 26, 0.1) 0px 1px 0px,
+    rgba(17, 17, 26, 0.1) 0px 8px 24px, rgba(17, 17, 26, 0.1) 0px 16px 48px;
+}
+
+.category-view.play-mode .card:hover {
+  box-shadow: 0 1px 3px 0 rgb(0 0 0 / 10%), 0 1px 2px 0 rgb(0 0 0 / 6%);
+}
+
+.flipped {
+  transform: rotateY(180deg);
+}
+
+.card-body {
+  display: grid;
+}
+
+.front-side,
+.back-side {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  backface-visibility: hidden;
+}
+
+.back-side {
+  transform: rotateY(180deg);
+}
+
+.back-side .card-img {
+  transform: rotateY(180deg);
+}
+
+.back-side .flip-div {
+  height: 30px;
+}
+
+.card-img {
+  width: 100%;
+  height: 150px;
+  object-fit: cover;
+  border-radius: var(--bs-card-border-radius) !important;
+  border-bottom-left-radius: 0 !important;
+  border-bottom-right-radius: 0 !important;
+}
+
+.card h5 {
+  margin: 0;
+}
+
+.category-view.play-mode .card-img {
+  height: 100%;
+  transition: 0.3s;
+}
+
+.game-on .card {
+  cursor: pointer;
+}
+
+.game-on .inactive-card {
+  opacity: 50%;
+  cursor: default;
+}
+
+.inactive-card .back-side {
+  display: none;
+}
+
+.flip-button {
+  justify-self: end;
+  display: block;
+  margin: 20px 20px;
+  margin-top: 0;
```

These properties can be merged:
```suggestion
  margin: 0 20px 20px;
```